### PR TITLE
Fix/bug

### PR DESCRIPTION
**Pull request by UmerFrooq**
Add a reference in the changelog

### CODE REVIEW

**Comment by tylerhaugen-stanley** [comment]
Not sure if you intended to open a new PR or not but i guess you'll want to close the other one now.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ### 1.4.1 (Next)
-* Your contribution here.
+* [#97](https://github.com/dblock/iex-ruby-client/pull/97): Update READme.md for histroical daily request date parsing issue - [@umer_frooq](https://github.com/UmerFrooq).
```
Comment (tylerhaugen-stanley) [inline CHANGELOG.md:2]: This should be 98 since you opened a new PR.

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ### 1.4.1 (Next)
-* Your contribution here.
```
Comment (tylerhaugen-stanley) [inline CHANGELOG.md:2]: We want to leave this in.

**Review by tylerhaugen-stanley — CHANGES_REQUESTED** [review]
Minor changes.

File: CHANGELOG.md (hunk 1 above)
Comment (UmerFrooq) [reply CHANGELOG.md]: sorry, it's first to contribute in the repo :)

**Comment by dblock** [comment]
Closed via #97, thanks for doing this @UmerFrooq!

**Comment by dblock** [comment]
For next time, check out CONTRIBUTING.md, it has some typical workflow suggestions for making changes and pull requests @UmerFrooq. It's totally confusing but eventually you'll get a hang of it :) Ask us anything here, too!

**Pull request closed without merging**